docs/UserGuide.adoc

```diff
@@ -96,219 +98,290 @@ Format: `tab TAB_NAME` +
+==== Editing a template: `edit`
 
-==== Adding a template: `add`
+Edits a template at the specified index. Only the name is editable. +
+Format: `edit INDEX n/NAME`
 
-Adds a new template to weme. The newly created template will not have any labels. +
-Format: `add n/NAME p/PATH` +
 Examples:
 
-* Add a template named “Drake” from ~/Downloads/Drake_hotline_bling.jpg +
-`add n/Drake p/~/Downloads/Drake_hotline_bling.jpg`
-
-==== Editing a template: `edit`
-Edits a template at the specified index. The list panel will become the editing area and display the selected template. You can add, remove, or move labels. A set of keyboard actions are available: +
+* `edit 1 n/Surprised Pikachu` +
+Edits the name of the 1st template to be `Surprised Pikachu`.
 
-* a: add a label
-* NUMBER: focus a label labelled NUMBER
-* ↓/j: move the currently focused label downwards
-* ↑/k: move the currently focused label upwards
-* ←/h: move the currently focused label to the left
-* →/l: move the currently focused label to the right
-* Delete/d: delete the currently focused label
-* Enter: finish editing
+==== Locating templates by name: `find` (_coming in v1.4_)
 
-Format: `edit INDEX`
+Finds templates whose names contain any of the given keywords. +
+Format: `find KEYWORD [MORE_KEYWORDS]`
 
 ****
-* Note: The current edit session will be aborted if another command is entered before the session is finished.
+* The search is case-insensitive. e.g `drake` will match `dRaKE`
+* The order of the keywords does not matter. e.g. `Pikachu Surprised` will match `Surprised Pikachu`
+* Only the name is searched
+* Only full words will be matched e.g. `sponge` will not match `spongebob`
+* Templates matching at least one keyword will be returned (i.e. OR search). e.g. `SpongeBob Patrick` will return both `Tired SpongeBob` and `Savage Patrick`.
 ****
 
+Examples:
+
+* `find Thanos` +
+Returns all templates containing `Thanos` in their names
+
+* `find Stonks Doge Pikachu` +
+Returns any templates having names `stonks`, `doge`, or `pikachu`
+
 ==== Deleting a template: `delete`
 
-Deletes the specified template from the template tab. +
+Deletes the template at the specified index. +
 Format: `delete INDEX`
 
-==== Locating templates by name: `find`
+==== Archiving a template: `archive`
 
-Finds templates whose names contain any of the given keywords. +
-Format: `find KEYWORD [MORE_KEYWORDS]`
+Archives the template at the specified index. +
+Format: `archive INDEX`
+
+==== Unarchiving a template: `unarchive`
+
+Unarchives the template at the specified index. +
+Format: `unarchive INDEX`
+
+==== Listing all archived templates: `archives`
+
+Lists all templates that are archived in the templates tab. +
+Format: `archives`
+
+==== Creating memes from templates: `use`
+
+Uses the template at the specified index to start a meme creation session. +
+Weme will enter the create tab and allow you to add text to the template. +
```

Comment:
Should standardise third person instead of first person. Hence user instead of you